tests/QueuedJobsTest/TestQJService.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace SilverStripe\QueuedJobs\Tests;
+
+use SilverStripe\QueuedJobs\Services\QueuedJobService;
+
+// stub class to be able to call init from an external context
+class TestQJService extends QueuedJobService
```

Comment:
These stubs should probably implement `SilverStripe\Dev\TestOnly` (I know they didn't before)